server/src/main/java/org/apache/iotdb/db/mpp/plan/planner/plan/node/PlanVisitor.java

```diff
@@ -93,76 +96,88 @@ public R process(PlanNode node, C context) {
 
   public abstract R visitPlan(PlanNode node, C context);
 
-  public R visitSeriesScan(SeriesScanNode node, C context) {
+  public R visitSourceNode(SourceNode node, C context) {
```

Comment:
What about `visitShowQueries`? It seems that you think `ShowQueriesStatement` is `query`, and we can also combine `show queries` with `limit` and `offset`.

Reply:
The implementation of `ShowQueriesOperator` does not consider limit pushdown, so there is no pushdown for `ShowQueries`. The Statement is judged in `LimitOffsetPushDown`, and it will continue to be optimized when it is `QueryStatement`.